tardis/plasma/properties/radiative_properties.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, plasma_parent):
 
     def get_g_lower(self, levels, lines_lower_level_index):
         if self._g_lower is None:
-            g_lower = np.array(levels.g.iloc[lines_lower_level_index],
+            g_lower = np.array(levels.g.ix[lines_lower_level_index],
```

Comment:
@wkerzendorf This was the problem. This originally used .iloc rather than .ix. It turns out that .iloc causes errors in Pandas 0.12 that were fixed in version 0.13. But .ix works fine.